website/project/metadata/aea-prereg-2.json

```diff
@@ -0,0 +1,480 @@
+{
+    "name": "AEA_Preregistration",
```

Comment:
Per discussion with @evomellor : as we add more prereg types, it may be worth spelling out organization names to avoid confusing overlap between disciplines.

(For example: wikipedia returns 11+ discipline-specific organizations that could plausibly be interpreted as "AEA")